applications/msai.md

```diff
@@ -0,0 +1,290 @@
+|    2a. | Create New Substrate Pallet            | [ 1. Create demo application](https://trello.com/c/LZ3c4K3T/3-create-basic-parachain-with-useful-api) New parachain with smart contract facility through creation of a dedicated Pallet                                                                                                                                                  |                                                                                                                                                        |
+|    3a. | Extend with Storage Pod API            | Basic data storage API (with OpenAPI specifications) built on top of OFW adding some of the functionality of Solid pods                                                                                                                                                                                                                  |
+|    4a. | Built in health tailored WASM contract | NFT smart contract default template-- Built in smart contract on the subtrate chain - written in Ink! or other webassembly language that supports NFTs and ERC721 level in addition to ownership authentication:                                                                                                                         |
+|    4b. | Add ID Validation to Pallet            | Integration of Identity validation mechanism :Built in method of assigning identity signers based on the described schema. THe schema is stored in a ink smart contract native to the Health substrate parachain. This built in smart contracts provides added value to NFT generated from users health data (As fake data has no value) |
+|    5a. | Extend Off chain worker Data Pallet    | Implementation Off chain worker encryption and decryption module that can authorize decryption when verified by a smart contract                                                                                                                                                                                                         |
+|    5b. | Data Validation on store               | Validation check every data stored on node against a hash must have a corresonding entry in template smart contract. In this first instance data storage will be assumed to be trusted as long as parachain node operator is trusted and hashes match the blockchain smart contract                                                      |
+|    6a. | Example with new api                   | Document Off chain encryption decryption API and create                                                                                                                                                                                                                                                                                  |
```

Comment:
An example of what? A frontend? A CLI client? A reference iplementation? Please specify further.

Reply:
A CLI  and possibly a GUI app if time permites client using the addition API of the substrate nodes.